packages/coordinator/tests/resources/test-helpers.ts

```diff
@@ -59,8 +61,16 @@ declare global {
 }
 
 if (!Array.prototype.remove) {
-  Array.prototype.remove = function<T>(this: Array<T>, elem: T): void {
+  Array.prototype.remove = function <T>(this: Array<T>, elem: T): void {
```

Comment:
This space may have been accidentally added

Reply:
removed